src/explore-education-statistics-admin/src/pages/release/data/ReleaseApiDataSetChangelogPage.tsx

```diff
@@ -28,11 +29,14 @@ export default function ReleaseApiDataSetChangelogPage() {
     data: dataSet,
     isLoading: isLoadingDataSet,
     refetch: refetchDataSet,
+    isError: errorFetchingDataSet,
```

Comment:
This isn't necessary as we can infer this when `dataSet` is undefined and loading has completed. In the error state, `dataSet` is not defined.

See later comment on why this check is beneficial type-wise as well.